src/key.rs

```diff
@@ -821,21 +877,43 @@ mod tests {
         use serde_test::{assert_tokens, Configure, Token};
 
         let mut rng = ChaCha8Rng::from_seed([42; 32]);
-        let priv_key = RsaPrivateKey::new(&mut rng, 64).expect("failed to generate key");
+        let priv_key = RsaPrivateKey::new(&mut rng, 1024).expect("failed to generate key");
```

Comment:
I guess this is an example that should probably be switched to a test vector key rather than trying to make one on-the-fly with a deterministic RNG.